src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -192,6 +192,10 @@ trait WorkflowComponent {
       sqlu"update WORKFLOW set status = ${newStatus.toString}, status_last_changed = ${new Timestamp(System.currentTimeMillis())}, record_version = record_version + 1 where status = ${currentStatus.toString}"
     }
 
+    def abortWorkflowsOfStatus(submissionId: UUID, status: WorkflowStatus): WriteAction[Int] = {
+      sqlu"update WORKFLOW set status = ${status.toString}, status_last_changed = ${new Timestamp(System.currentTimeMillis())}, record_version = record_version + 1 where status = ${WorkflowStatuses.Queued.toString} and submission_id = ${submissionId}"
```

Comment:
It should be aborted, but yeah, I changed this to be a bit more generic and committed a halfway finished version of it for some reason. Nice catch.